File: hw6/Circle.java
import hw6.Shape;


public class Circle extends Shape{
    private int radius;
    public Circle(String color, int r){
        super(color);
        radius = r;
    }
    public void print(){
       double area = 0;
       String color;
       area = get_area();
       color = get_color();
       System.out.println(color + " circle, radius " + radius + ", area " + area);
    }
    public double get_area(){
        double area = 0;
        area = 3.141592653559 * radius * radius;
        return area;
    }
}

File: hw6/Circle.cpp
#include    "Circle.h"

Circle::Circle(const string & color, int inp):Shape(color){
    radius = inp;
    cout << "Circle construct." << endl;
}

Circle::~Circle(){
    cout << "Circle destruct." << endl;
}

void Circle::print(){
    double area = 0;
    string color;
    area = get_area();
    color = getcolor();
    cout << color <<" circle, radius " << radius << ", area " << area << endl;   //green circle, radius 10, area 314.159
}

double Circle::get_area(){
    double area = 0;
    area = 3.14159265359 * radius* radius;
    return area;
}